packages/react-provider/src/ws.ts

```diff
@@ -14,29 +15,25 @@ export class WS {
 
   connect(clientKey: string): void {
     const url = `${this.url}/?clientKey=${clientKey}`;
-
-    if (typeof WebSocket) {
-      this.connection = new WebSocket(url);
-      this.initiateListener();
-    }
+    this.connection = new WebSocket(url);
```

Comment:
do we not need to check for Websocket? guessing if the react is run on server-side, it will fail, no?

Reply:
id rather have it fail than silently do nothing